src/backends/pgfplots.jl

```diff
@@ -10,6 +9,134 @@ function _initialize_backend(::PGFPlotsBackend; kw...)
   # TODO: other initialization
 end
 
+_pgfplots_get_color(c) = "{rgb,1:red,$(float(c.r));green,$(float(c.g));blue,$(float(c.b))}"
+
+function _pgfplots_get_linestyle!(kwargs, plt)
+    linestyle = plt[:linestyle]
+    if linestyle == :dash
+        kwargs[:style] *= "dashed,"
```

Comment:
It might be slightly nicer to look at if you used a constant map:

```julia
const _pfg_linestyles = KW(
  :dash => "dashed",
  ...
)

function _pgf_get_linestyle!(kwargs, d)
  ls = d[:linestyle]
  if haskey(_pgf_linestyles, ls)
    kwargs[:style] *= _pfg_linestyles[ls]
  end
end
```

You could also accumulate styles in a vector and then `join(vec, ",")` them.